### DISCUSSION TITLE
🍁 Seasons and Weather

### DISCUSSION THREAD

**Discussion by adamhannigan** [sunflower-land/sunflower-land#4852] [Ideas]
# 🍁 Seasons & Weather

Introducing Seasons, a time based system that affects resource, farming & special events.

Every **7 days**, a new season will occur in Sunflower Land providing different crops, flowers, fish & more:

* Summer
* Spring
* Winter
* Autumn

Watch out for the special events on the calendar! These can be times of prosperity or disaster:

* Storms 🌩️ 
* Tsunamis 🌊 
* Bountiful Harvests 🌻 
* Bloodmoon 🌔 

_We will need to change our current "Season" terminology we use -> Collection._
_E.g. "Bull Run Collection", "Dawn Breaker Collection"_

### Player Experience

Overall, we want this mechanic to **amplify scarcity** and the marketplace trading. This makes gameplay more **strategical**. Other emotions we are tapping into:

1. Risk aversion - should I prepare for upcoming disasters or shortages
2. Variability - always fresh content and events popping up
3. Anticipation - Making 'bets' and choices on upcoming unknown events
4. Curiosity & Experimentation - Different flowers & fish found based on the season

## Themes

Depending on the season will determine a biome for your farm and the plaza. For example, during Winter there will be snow caps on your resources and buildings.

## Crops & Fruit

Depending on the season, different seeds will be available for purchase and sale at Betty's market. Players must strategically plant and hoard crops for their needs. Each season will have 10 Crops & 4 Fruit that can be grown. Some crops/fruit will appear in multiple seasons, some will only appear in one.

<img width="1564" alt="Screenshot 2024-12-03 at 3 01 10 PM" src="https://github.com/user-attachments/assets/870260f2-e620-4f7f-9d53-bb70146dc343">

Seasons **should not be perfectly balanced with each other.**

_If a seed is in off season, it cannot be planted._ _Sunflowers will be available year round._

<table><thead><tr><th width="153">Name</th><th width="100">Time</th><th width="93">Spring</th><th width="100">Summer</th><th width="102">Autumn</th><th>Winter</th></tr></thead><tbody><tr><td>Sunflower</td><td>1 min</td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td></tr><tr><td>Potato</td><td>5 min</td><td></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td></tr><tr><td>Rhubarb <span data-gb-custom-inline data-tag="emoji" data-code="1f539">🔹</span></td><td>10 min</td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td><td></td><td></td><td></td></tr><tr><td>Pumpkin</td><td>30 mins</td><td></td><td></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td><td></td></tr><tr><td>Zucchini <span data-gb-custom-inline data-tag="emoji" data-code="1f539">🔹</span></td><td>30 mins</td><td></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td><td></td><td></td></tr><tr><td>Carrot</td><td>1 hr</td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td><td></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td><td></td></tr><tr><td>Yam <span data-gb-custom-inline data-tag="emoji" data-code="1f539">🔹</span></td><td>1 hr</td><td></td><td></td><td></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td></tr><tr><td>Cabbage</td><td>2 hr</td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td><td></td><td></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td></tr><tr><td>Broccoli <span data-gb-custom-inline data-tag="emoji" data-code="1f539">🔹</span></td><td>2 hr</td><td></td><td></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td><td></td></tr><tr><td>Soybean</td><td>3 hr</td><td></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td><td></td><td></td></tr><tr><td>Hot Pepper <span data-gb-custom-inline data-tag="emoji" data-code="1f539">🔹</span></td><td>4 hr</td><td></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td><td></td><td></td></tr><tr><td>Beetroot</td><td>4 hr</td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td><td></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td></tr><tr><td>Cauliflower</td><td>8 hr</td><td></td><td></td><td></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td></tr><tr><td>Parsnip</td><td>12 hr</td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td><td></td><td></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td></tr><tr><td>Coffee <span data-gb-custom-inline data-tag="emoji" data-code="1f539">🔹</span></td><td>12 hr</td><td></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td><td></td><td></td></tr><tr><td>Eggplant</td><td>16 hr</td><td></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td><td></td></tr><tr><td>Onion <span data-gb-custom-inline data-tag="emoji" data-code="1f539">🔹</span></td><td>20 hr</td><td></td><td></td><td></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td></tr><tr><td>Corn</td><td>20 hr</td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td><td></td><td></td></tr><tr><td>Turnip <span data-gb-custom-inline data-tag="emoji" data-code="1f539">🔹</span></td><td>24 hr</td><td></td><td></td><td></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td></tr><tr><td>Radish</td><td>24 hr</td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td><td></td><td></td></tr><tr><td>Wheat</td><td>24 hr</td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td></tr><tr><td>Kale</td><td>36 hr</td><td></td><td></td><td></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td></tr><tr><td>Artichoke <span data-gb-custom-inline data-tag="emoji" data-code="1f539">🔹</span></td><td>36 hr</td><td></td><td></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td><td></td></tr><tr><td>Barley</td><td>48 hr</td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td><td></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td><td></td></tr></tbody></table>

### Fruit

_There is a shortage of fruit in Winter_ :snowflake:

| Name      | Time  | Spring               | Summer               | Autumn               | Winter               |
| --------- | ----- | -------------------- | -------------------- | -------------------- | -------------------- |
| Tomato    | 2 hr  | :white\_check\_mark: | :white\_check\_mark: | :white\_check\_mark: |                      |
| Lemon     | 4 hr  | :white\_check\_mark: | :white\_check\_mark: |                      |                      |
| Chestnut     | 4 hr |                      |                      | :white\_check\_mark: | :white\_check\_mark: |
| Blueberry | 6 hr  | :white\_check\_mark: | :white\_check\_mark: |                      |                      |
| Starfruit 🔹 | 6 hr  | :white\_check\_mark: |                      | :white\_check\_mark: |                      |
| Orange    | 8 hr  | :white\_check\_mark: | :white\_check\_mark: | :white\_check\_mark: | :white\_check\_mark: |
| Apple     | 12 hr |                      |                      | :white\_check\_mark: | :white\_check\_mark: |
| Banana    | 12 hr |                      | :white\_check\_mark: | :white\_check\_mark: |                      |
| Coconut 🔹   | 12 hr | :white\_check\_mark: | :white\_check\_mark: |                      |                      |

## Tutorial

When a player begins the game, they will start in Summer mode (regardless of when they start). After 5 days, the will fall into the actual cycle.

## Calendar

Players will be able to view the upcoming seasons in Codex Calendar.

<img width="264" alt="Screenshot 2024-12-03 at 3 02 11 PM" src="https://github.com/user-attachments/assets/c1332ec3-a09d-4551-9d06-b1505f0c1b9d">

**Special events** will also be marked on the calendar, but players **will not know what they are.** ❓ 

The HUD will also include an indicator of what season it is in the game.

<img width="346" alt="Screenshot 2024-12-03 at 3 02 54 PM" src="https://github.com/user-attachments/assets/517a4e41-b977-4df2-a976-4849e341be0b">

## Special Events

We will  build on top of the special events we already have in the game (Fish Frenzy, Full Moon & Double Delivery).

Special events can be good or bad for players, depending how they prepared. Certain events can only occur during certain seasons. For instance, Tsunami's only occur in Summer.

<table><thead><tr><th width="194">Event</th><th width="117">Spring</th><th width="136">Summer</th><th width="118">Autumn</th><th>Winter</th></tr></thead><tbody><tr><td>Fish Frenzy <span data-gb-custom-inline data-tag="emoji" data-code="1f41f">🐟</span></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td></tr><tr><td>Full Moon <span data-gb-custom-inline data-tag="emoji" data-code="1f315">🌕</span></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td></tr><tr><td>Double Delivery <span data-gb-custom-inline data-tag="emoji" data-code="1f381">🎁</span></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td></tr><tr><td>Tsunami <span data-gb-custom-inline data-tag="emoji" data-code="1f30a">🌊</span></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td><td></td><td></td><td></td></tr><tr><td>Great Freeze <span data-gb-custom-inline data-tag="emoji" data-code="2744">❄️</span></td><td></td><td></td><td></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td></tr><tr><td>Tornado <span data-gb-custom-inline data-tag="emoji" data-code="1f32a">🌪️</span></td><td></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td><td></td></tr><tr><td>Bounty Harvest <span data-gb-custom-inline data-tag="emoji" data-code="1f33b">🌻</span></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td><td></td><td></td></tr><tr><td>Insect Plague <span data-gb-custom-inline data-tag="emoji" data-code="1f41c">🐜</span></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td><td></td><td></td></tr><tr><td>Rain <span data-gb-custom-inline data-tag="emoji" data-code="1f327">🌧️</span></td><td></td><td></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td><td><span data-gb-custom-inline data-tag="emoji" data-code="2705">✅</span></td></tr></tbody></table>

**Fish Frenzy** :fish:

* 2x Fishes
* 24 hour event

**Full Moon** :full\_moon:

* Catch Marine Marvels
* 24 hour event
* Nightshade special seeds can be bought/planted - Duskberry, Grimroot, Lunacress

**Double Delivery** :gift:

* Double rewards for deliveries
* 24 hour event

**Tsunami** :ocean:

* One off event
* "Floods' all growing crops and doubles remaining time to grow
* Craft a wall to protect from floods, expires after 60 days

**Great Freeze** :snowflake:

* 24 hour event
* Half of plots are frozen for 24 hours
* Craft X to protect from freeze, expires after 60 days

**Tornado** :cloud\_tornado:

* One off event
* All of crops & fruit removed
* Craft a wind breaker to protect, expires after 60 days

**Bounty Harvest** :sunflower:

* 24 hour event
* Extra +1 Harvest on all crops and fruit

**Insect Plague** :ant:

* 24 hour event
* -0.5 harvest on all crops

**Sunshower** :cloud\_rain:

* 24 hour event
* Crops 2x speed

## Weather Prevention

Are you looking to avoid de-buffs and lost of crops during special events? You will need to craft certain equipment which will expire after a number of days.

This will typically consist of gathered resources + animal materials.

A player must decide if they want to risk it (not spend resources) or play it safe and prepare for an event.

## Mechanic Updates

So what happens if you don't have access to certain resources to play the game? We will ensure the game is balanced & enjoyable all year around!

We will be adapting the mechanics so Sunflower Land is always playable and users can progress throughout the game. 
There are some key features that will be tweaked.

### Flowers

Introduce 4 extra breeds of flowers (with multiple variations), that can only be grown in each respective season.

### Fishing

Introduce 8 more fish and include weather in the chances/requirements to catch fish (including existing).

⛔  Remove dawn/dusk logic & replace with seasonal catching

### Animals

Wheat will remain for every season, as it is used as the staple feed for animals. Barley and Corn are more scarce to find.

The team are preparing some animal improvements, especially around feed and will address in a separate proposal. 🙏 

### Recipes

We will need to introduce more recipes which are seasonal focussed (i.e. all ingredients can be found in a specific season).

We will introduce 6 new '**seasonal recipes**' which will change requirements based on the season. For instance, a seasonal salad in Summer would require Zuchinni & Eggplant. A Seasonal Salad in Winter would require Parsnip & Cauliflower.

### Composting

Composters only ask for resource in a given season.

## Deliveries

80% of deliveries only request seasonal related items. However, the other 20% will request resources outside of the season. This will put demand on out of season resources.

### Chores

Chores will only include seasonal related content.

### Greenhouse

The greenhouse stays the same (crops grown inside are not weather dependent).

## Volcano Island

Worth noting the upcoming season we will launch Volcano island, say tuned for another proposal.

# Conclusion

Thank you for taking the time to read this proposal. If this is your first time, we would deeply appreciate any feedback or suggestions. Although we don't respond individually, we read every comment and update the proposal based on your ideas.

Thanks! 🤗

**Comment by TheDcol** [comment]
Very excited! Always wanted weather/seasons in the farming game to play a part :)

I'd like to start a discussion on whether players should ALL be on the same season/weather OR whether it would be interesting to have different sections of players be on different seasons, rotating each week.

A lot more will have to be considered, such as seasonal races and chores (given the disabled crops), but I wonder if it would make this more epic (or not)

**Reply by Xeko-92** [reply]
How would that work with deliveries if people are supposed to have 80% of the deliveries from the current season? Would that mean people have different deliveries from each other?

Also, would that mean seasons are in a random order rather than a specific order?

**Reply by adamhannigan** [reply]
@Xeko-92 Currently seasons are in the same order + same seasons for everyone.

There is the option/idea that players have different seasons to encourage collaboration (not included in this proposal)

**Reply by LucasAG95** [reply]
@adamhannigan  I think it would be interesting for half of the players to be in the south and half in the north, so we would have 2 seasons occurring together. While it's summer in the North, it's winter in the South

**Comment by snorri-snorri** [comment]
Will the special events begin at 0000 UTC and if so will that put EU timezones at any disadvantage if we're not awake to react to the event?

**Reply by snorri-snorri** [reply]
Copied from discord, posted by LaMouette;

Hey! I just red the proposal for Season weather and all. That's sounds really interesting! 
I have a question though about the "one off" events (tornado/tsunami): will they always happen on reset? If yes that puts people who are sleeping at that time at a disadvantage (specially with tornado that removes crops/fruits) because people plant night-long crops and if removed at reset you loose a lot more than people that are awake at that time and can just plant nothing low growth time crops. 
If it happens on reset it's always the same people that have "an advantage" of loosing less/anticipating, without the "need" to maybe build the wind breaker or something.
It's specially the tornado because it removes crops. But it could also be an issue for tsunami (doubling night-long time growth and you loose half the day but that's not the case if you are awake/plant potatoes).

**Reply by adamhannigan** [reply]
Yes, special events will be at UTC 0000, same time for everyone.

There isn't much you can do to react to an event - instead it is all about 'preparing' for an event days in advance.

There are 8-9 events. It wouldn't be the wisest strategy for instance to harvest everything before UTC midnight ticks over, as chances are it isn't the tornado/tsunami 😅 

Once we get a better gameplay usage, we will ensure that timezones don't greatly affect players 🤗

**Reply by Xeko-92** [reply]
If all of your crops get taken out and you can replant them right away, there's a big difference if you're online right at that time or if you need to wait 6 hours.

**Reply by adamhannigan** [reply]
@Xeko-92 Gotcha make sense now 👍 I need to think more around this

**Comment by snorri-snorri** [comment]
Also, if a new player starts the game, isn't this too much for their farm? 

If they need to start crafting stuff what are they using to craft? Can't be animal produce as even veterans of the game are struggling with animals, they won't be in a position to trade on the marketplace.

**Reply by adamhannigan** [reply]
Onboarding will definitely play a huge part in this.

For the first X days, all new players will be in the same season (summer). After that, they will fall into the normal cycle.

We will ensure that progression, craftables, etc all cater to resources a new player who is learning has access to.

**Comment by Xeko-92** [comment]
I'm going to call current seasons 'leagues' instead of 'collections'. I don't feel 'collections' makes sense for what they are.

**Reply by adamhannigan** [reply]
We are definitely open to some more names for this 💯 I would be interesting to hear if there are any other possibilities as well.

**Reply by Dionis404** [reply]
League, battle, championship, tournament

**Reply by snorri-snorri** [reply]
Chapter 🙃

**Comment by Xeko-92** [comment]
Any thought about increasing the seed limit so we can stock up on seeds during off seasons? This update feels like it might be a nerf to restocking if we can only plant certain things, some which we might not care about as much as others.

**Reply by adamhannigan** [reply]
We will investigate this 👍 We do need consider what happens to off-season seeds/crops - if you restock Betty's market are you also restocking the non visible seeds?

If so, this new system would be a bigger benefit as overall you are getting more seeds than before (since there are new crops + fruits)

**Comment by Xeko-92** [comment]
Another concern that LaMouette brought up in discord is that this is a ton of new crops and fruit. What tends to happen is that things are released and there is either A) little use for them or B) no use.

Examples: sunstone, community coin still have no use.

Another example is tomatoes and lemons had no use for the entire season in which they were launched.

There are others too that have very marginal use, I won't bring those up here as they might take away from the discussion.

And to note, in my opinion, being asked for in a delivery isn't a use case. You can have deliveries without those items. You can't have lemon juice without lemons.

**Reply by adamhannigan** [reply]
Definitely all fair critics 👍 

It is something recently I feel we are getting better at (but still improving). Every resource would have a tailored use in this system - especially with our marketplace launch we need to ensure demand for everything.

**Reply by Thebyber** [reply]
I put it in the animal tweaks proposal, I think it's the perfect time to renew all the recipes in the kitchens, there are many recipes that are unbalanced.

For example: there are many foods in the deli that take 1 day to gain 400-600 exp (Fermented carrots-Sauerkraut).. instead you make a fried tofu and you gain more experience in just one hour.

The recipes with mushrooms are very low in experience for a resource that is obtained in a very long time, you get 1 mushroom in 18 hours, the mushroom soup needs 5 mushrooms to gain 70 exp... it's ridiculous, currently it only works for Bert's missions.

All the cakes give very little experience for the time they take to make, for example I only make cheesecakes... the rest of the cakes don't exist unless you order them in a delivery.

I know it is difficult to renew all the meals but I think that at some point it should be done. You can also integrate a new kitchen where exclusive seasonal recipes are prepared.

**Reply by adamhannigan** [reply]
Thanks @Thebyber We are definitely looking at improving these recipes.

I will preface that we've purposely got some unbalanced recipes - part of the game is figuring out what to cook and why (there are a few duds in there only really used for missions).

That said we will address some outliers 👍

**Reply by Alucard0389** [reply]
What is if the cooking recipe gives exp depending on the level from the player? This could also make old recipes interesting again.

**Comment by heatedlime** [comment]
Crop machine should also be able to grow crops year round, regardless of the season

**Comment by Dionis404** [comment]
The basic idea is very interesting, and creates some kind of artificial scarcity of resources that will stimulate the market.

Here are some remarks and my thoughts.

- Too many new crops (already now we have useless crops that are not emphasized by many people, for example, radish has no such active use as wheat, which greatly reduces its demand).

- Fractional season - each faction only has the ability to grow their crops in their own season.
SUNFLORIANS - They can grow X crop in the summer (12 hours)
NIGHTSHADES - Can grow Z crops in the fall (12 hours)
GOBLINS - Can grow Y crops in the winter (12 hours)
BUMPKINS- Can grow H crops in the spring (12 hours )        
This gives the opportunity to stimulate the market more strongly due to the fact that only one faction can grow crops that will be demanded later in deliveries and cooking.

- Animals - Feed less in the summer and feed more in the winter
 
 **Questions:**
 What happens to the crop if it doesn't have time to grow during its season?

 How will the seasons affect resource extraction?

**Reply by adamhannigan** [reply]
Some great ideas 👌 

- Do you have ideas that can increase the utility of certain crops inside of our current system?
- We've wanted to explore faction specific buffs + resources. This definitely feels like a good use case 👀 
- Animals. I love this idea 👌 

What happens to the crop if it doesn't have time to grow during its season - As long as you plant it in the season, you can harvest it whenever.

How will the seasons affect resource extraction - I imagine more people would be hoarding + harder for OP crop extractors (i.e. someone with all the cabbage buffs) to impact the economy. The player/economy driven side can be hard to predict though!

**Reply by Dionis404** [reply]
> Do you have ideas that can increase the utility of certain crops inside of our current system?

The easiest option is to add more pet food. (with different amounts of experience)
And make a division of this food by animal.

For example:
- Chickens: sunflower, cabbage, wheat are used
- Sheep: potatoes, beets, radishes
- Cows: Pumpkin, soybeans, Barley

Of the more complex ideas, are changes to the cooking system. 
I Previously wrote about this but after the GitHub lockdown the entry was deleted, if you are interested I can describe the idea again.

**Comment by Thebyber** [comment]
It seems like a pretty good idea to me, I think that the seasons within the game can give a freshness and a lot of strategy but there are many details that need to be clarified.

Special Events
If the calendar says that there is a negative event such as the tsunami or tornado on day 3... if on day 2 at the last minute I have the crops harvested... on day 3 I would have the crops available and without penalty since it would have happened the tsunami or tornado.

Animals
This is a complicated issue, currently animals are unattainable for new players and intermediate players... only players who invest a lot are able to maintain their diet.. Likewise when introducing these proposals, new foods will have to appear.

Fishing
What will happen to the guaranteed catches like the anchovy that requires carrots? Will there be new fishing guaranteed for anchovy? Also, why not put a higher fishing drop for certain fish in different seasons? For example, there is a greater chance of finding a sunfish in summer than in winter.

Factions
Another no less important detail, what about the factions? Can factions order crops or fruits from other seasons? It may be interesting to include a faction calendar in which players can plant resources in other seasons
An example: Sunflorians can plant kale in spring, artichoke in summer,
Bumpkins can plant artichoke in spring and kale in winter.

**Reply by Dionis404** [reply]
> Factions
> Another no less important detail, what about the factions? Can factions order crops or fruits from other seasons? It may be interesting to include a faction calendar in which players can plant resources in other seasons
> An example: Sunflorians can plant kale in spring, artichoke in summer,
> Bumpkins can plant artichoke in spring and kale in winter.

I'm also in favor of factions having their own season and their own harvest.
But it should be unique, which will stimulate the market more.

**Reply by Thebyber** [reply]
Yes, the idea is to rotate the mechanics of planting, for example kale, between the different factions... if a specific crop is planted for a faction it can create bad vibes.

Another point that I forgot, when introducing the seasons... could new resources appear on the beach to be dug?

**Reply by adamhannigan** [reply]
Thanks for the feedback 👌 

To clarify, the calendar will not say whether it is a good or bad event. Just that an event is coming.

Yes you are right, you could "prepare" for a tornado and remove crops. There is 1/8th chance of being that special event though. We will investigate some ways to mitigate this workaround.

Fishing - the guaranteed fish will still remain for every season. We are just adding some bonus fish to catch in different seasons and tweaking current drop rates as you said (e.g. Sunfish in Summer)

Factions - similar above in @Dionis404 comment I think it's worth exploring. I do want to see some crops **only available** in a given season though to create scarcity. However, I am open to their being extra 'faction crops' available that serve this exclusive perk purpose. We just need to be wary of Bumpkin traitors swapping factions 😅 

Beach - Yes, that makes sense as well 👌 The seasons should affect every mechanic in the game and give it freshness 😄 We will include this in the proposal 🙏

**Reply by Thebyber** [reply]
> Beach - Yes, that makes sense as well 👌 The seasons should affect every mechanic in the game and give it freshness 😄 We will include this in the proposal 🙏

The objects would be different depending on the season and can also affect new events.

For example, in the tsunami event it could wash some treasure onto the beach and with the tornado event there could be some holes already dug where the player has to dig less to find treasures.

**Reply by ThamaHalgravs** [reply]
i would love to see a frozen beach, with maybe a fishing hole to still give the chance to fish!

**Comment by snorri-snorri** [comment]
Will new players have a plenty of new player friendly recipes to cook with the seasonal crops? 
Is the grow time of seasonal crops something being considered for new players and how the game feels when someone starts playing? 

Feels like you would need a load of new recipes catering towards both new and older players that are designed using seasonal crops and these recipes would need to be available at season launch, not 3 months later.

Can't have new players starting and struggling to cook food and level up.

**Reply by Dionis404** [reply]
Leveling up and cooking for new players is a very sore subject that needs a separate discussion.

**Reply by adamhannigan** [reply]
@snorri-snorri Definitely one of the biggest challenges with this implementation.

I would suggest that during tutorial island (first island), players are always in 'Summer' mode. All of the onboarding and initial progression is tailored around that season 👌

**Comment by Thebyber** [comment]
I have seen that there are no new fruits for winter... have you thought about introducing chestnuts, which are produced in autumn-winter?

**Reply by adamhannigan** [reply]
Nuts sounds like a great introduction 👌 Would open up a lot of recipes - just need to make sure no one has any allergies!

**Comment by Harlen98** [comment]
I thought for balance between whales and newer players. That natural events such as tsunamis and tornadoes in their path will wear down buildings, as if deteriorating them. Thus, from time to time, resources are spent on repairing them, even if it is a small amount, which would encourage greater burning of these. Also, these damages increase depending on the current island of each player (example: Players on a basic island, would have natural events the same as the others but these would not affect the buildings. And as you advance to petal island or desert island they would be increasing the damage scale). What do you think?

**Reply by Dionis404** [reply]
That's a good idea, except I would remove that mechanic from the base island altogether ( it should completely turn into a training island)
This would be a good indicator of the complexity of the gameplay depending on pumping farms

**Reply by adamhannigan** [reply]
Nice suggestion @Harlen98 👏 

Involving the buildings adds a natural scaling form of risk. Long term players will have more buildings (and more expensive ones) so it becomes a bigger risk for them 👌

**Reply by Harlen98** [reply]
Take the idea boss 😅, it's to put it into practice 👍

**Reply by Xeko-92** [reply]
Curious as to which buildings would be damaged and how the damage would actually affect the buildings?

**Reply by Harlen98** [reply]
Honestly, I would like to know the scale of damage to buildings as you progress from island to island 👍

**Reply by Xeko-92** [reply]
I was asking you what your idea was about damaged buildings. What does a damaged building mean?

**Reply by Harlen98** [reply]
I only mentioned the idea of ​​deteriorating buildings to make better use of resources by having to repair them. How they use it depends on the developers 😅. (By damaged building I mean damage caused by tsunamis or tornadoes)

**Comment by THINKTRONIK** [comment]
Wow ! 
Hi there, my first comment on github 😅
This Seasons and Weather brainstorm looks interesting.

For the faction traitors part... maybe Dev can add a restriction to player that quit faction have to wait until next season to join another faction OR limit amount player can swap faction in a month 🤔 

Surely this going to happen because those crops requested by faction donation always 🚀

**Reply by adamhannigan** [reply]
That's some food feedback 👍 

Using seasons as a time measurement + restrictions should be an easy concept for players to grasp

**Comment by ThamaHalgravs** [comment]
how this will impact che crop machine?

**Reply by adamhannigan** [reply]
The crop machine would only support the current seasonal crops.

Only the green house is not impacted.

**Reply by heatedlime** [reply]
That should really be revisited. Crop machine should be able to plant what it can, regardless of season. That really nerfs crop machine otherwise

**Comment by ThamaHalgravs** [comment]
I like a lot this new weather season, however, i was thinking about restocking, a season lasts 7 days and personally i actually restock once a day (rarely 2), some people even more, so since some seeds cannot be planted during certain seasons, there's a way to like stock more seeds?  Taking for example pumpkins, we stock up to 480 seeds, but that's about 3 restock, wich i belive most people do it in less than a week and we have to wait 3 weeks (21 days). So i mean that every time we restock forcefully some basic seeds are gonna be wasted because we have to wait too many days for the right season to plant them again

**Reply by adamhannigan** [reply]
Thanks for the feedback. Yes we will definitely factor in stock amounts to ensure players still get the same seeds as they currently do.

**Comment by Xeko-92** [comment]
What happens to crops/fruit that are planted in one season but don't complete until the following season where they aren't usable? Will they still be able to be harvested?

**Reply by adamhannigan** [reply]
Yes, you can still harvest them. As long as they are planted within the season.

**Comment by Rudolph-the-Bumpkin** [comment]
Can we use Sunstones to control the weather?
Maybe using sunstones to change it and be able to plant whatever you want for x time.

It definitely needs some utility

**Reply by adamhannigan** [reply]
We've got some plans for Sunstones we will share soon!

**Comment by Xeko-92** [comment]
I was talking with CoDefender and he had another idea for a resource sink for the weather-prevention tools that I thought was good. Maybe incorporating sand from digging into sand bags to hold things down.

The only issue I can see with it is that it would force people to dig since you can't trade sand. I still liked the idea though so I thought I'd share it.

**Comment by Rudolph-the-Bumpkin** [comment]
Regarding the Calendar, it would be a great way to include "Sunflower Land Community Events" to add more sinks / mechanics to the game. I know it's not the "core feature" this season, but there is so much potential on it! I really think we should extract the most from it!

It's common in farm simulators to have events like "dancing bowl", "animal competition", "cooking competition" etc. What if we also add some of those events into the Calendar? I know it isn't the core content, but could be related to it if the requirements meet the seasons criteria. 

Plus, it would make the game a lot more fun!

1) "Food Hype": A recipe is trending in Sunflower Land. A specific dish will be available as a gift to every NPC in the game during the entire day, for extra friendship points. (crops sink)
2) "Shrimp wave": A specific fishing event where players can catch shrimp. Increased chance Chum: crabs (wood / stone / crabs sink)
3) "Greedy Metabolism": Potion house tickets cost 9x more, but rewards are 10x better. (coins sink)
4) "Highlander Dimension": Portal goals get extremely difficult, but also gives extra rewards such as keys or more faction points. It would be a great way to fund Portals Builders. (SFL sink)
5) "Charity": Donate crops, clothes and toys to people in need for extra reputation points. Homeless guy comeback? We could track reputation points through friendship points with Homeless Guy (crops and craft box items sink) - we can even link them to the real world if we use SFL from treasury to "buy" the donations, so we would have a dollar fund.
6) "Tailor visit": A world wide famous Tailor is in Town! Visit him, deliver the resources needed and craft an exclusive wearable for your Bumpkin! (could use arts made by community for this). Community could fund artists this way
7) "Artisan Lessons": A great Bumpkin Artisan is in Town! Visit him, deliver the resources needed and craft an exclusive decoration for your farm! Or even better, a new and exclusive craftbox recipe! (could use arts made by community for this). Community could fund artists this way
8) "Bumpkin Monument": From time to time, players need to donate craft box resources to contribute to a new city building. This could act as a cooperative goal for the entire server. Once the goal is achieved (should last for days to be engaging), a new city building would be added to Plaza - it could be either decorative or have an utility. (craft box resources / minerals sink)

The mayor could be the NPC that gives us this little side quest and could also increase reputation points through it's own friendship bar.

Example: City Fountain

The same concept could be added to Faction Houses too.

Reference for friendship points as a guideline to reputation system: https://github.com/sunflower-land/sunflower-land/discussions/4976#discussioncomment-11612375

I feel like we have this great opportunity to add depth to the game using those calendar events. Can't wait to see what the future is reserving for us!

**Comment by Xeko-92** [comment]
Chapter is okay. It's better than collection.

**Comment by brian-man** [comment]
Event idea...
Eclipse
Effect: Swap time of day for Dawnlight and Dusktide.
